infra/gcb/builds_status.py

```diff
@@ -107,21 +98,14 @@ def execute_with_retries(request):
       raise
 
 
-def main():
-  if len(sys.argv) != 2:
-    usage()
-
-  projects_dir = sys.argv[1]
-
-  credentials = GoogleCredentials.get_application_default()
-  cloudbuild = gcb_build('cloudbuild', 'v1', credentials=credentials)
-
+def update_build_status(
+    cloudbuild, projects, build_tag, status_filename='status.json'):
```

Comment:
let's remove the default arg (status_filename) and always have the caller pass this.

Reply:
Done!